modules/account-lib/test/unit/mpc/tss.ts

```diff
@@ -172,21 +168,22 @@ describe('TSS EDDSA key generation and signing', function () {
       const path = `m/0/0/${index}`;
       const derive1 = MPC.deriveUnhardened(commonKeychain, path);
       const derive2 = MPC.deriveUnhardened(commonKeychain, path);
+      const derivedPk = Buffer.from(derive1, 'hex').slice(0, 32).toString('hex');
```

Comment:
@pranavjain97 when updating SDK in platform, we'll also need to extract the public key like this.

Reply:
Yeah makes sense. I think extracting the public key on platform is only done here-
https://github.com/BitGo/bitgo-microservices/blob/824d140eee92168e8c47eeab12f6800d2bb9af18/packages/wallet-platform/app/controllers/api/v2/coins/features/thresholdSignatureScheme.ts#L83